### ISSUE TITLE
Stage Recognition Failure in several stages - Auto I.S.

### ISSUE THREAD

**Issue by whylorainer** [MaaAssistantArknights/MaaAssistantArknights#7532]
### Before raising the issue...

- [X] I understand that Issues are for feedback and problem solving, not trolling the comments section, and will provide as much information as possible to help solve the problem.
- [X] I filled in a short, clear title so that developers could quickly identify the general problem when going through the issue list. Instead of "Some suggestions", "Stuck", etc.
- [X] I have checked the latest beta update and there is no mention of the bug being fixed.

### Description

When using Auto I.S. to farm Mizuki I.S., MAA will often output "Stage recognition error" and stop working, no longer deploying operators or interacting with the map, leading to map failure.

### Log files

The error happens all across the file.
[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13635047/gui.log)
[mutual_aid_failure_asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13635049/mutual_aid_failure_asst.log)

### Screenshots

These screenshots shows MAA not deploying operators and errors outputted in the software.

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/42009605/14dc449c-2ccf-4959-ae4d-3ba1827159b6)

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/42009605/fbcb0928-f6e4-41f7-8fbe-a032bec9eb4b)
show

### Anthing else?

LDPlayer9 used with the following configurations:
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/42009605/1d4cbee5-854f-4946-aa1c-42386173106c)

**Comment by Manicsteiner** [comment]
It was updated in task.json but not pushed to users. Please try to use the latest [ci builds](https://github.com/MaaAssistantArknights/MaaAssistantArknights/actions/workflows/ci.yml)/[beta builds](https://github.com/MaaAssistantArknights/MaaRelease/releases), or get /resource/global/YoStarEN/resource/tasks.json from the repository and overwrite the local file, or wait for next release.

**Comment by Manicsteiner** [comment]
> @Manicsteiner I'm sorry, but what kind of push are you talking about? In the past 4 days there have been no pushes regarding the I.S. for the YostarEN client.
No, my local 4.28.0-beta client still hasn't received this update for I.S. mode, which is a common thing. I switch to my manual build every time I encounter an update like this.

**Comment by Constrat** [comment]
Nevermind I see the problem. The IYostar clients I.S. were updated. My bad. Now there are new maps. It's understandable they haven't been updated yet.

**Comment by Constrat** [comment]
Fixed in e98ae16b39f64974c07a4fffbef0d781d44efd1e
